tests/virt_autotest/virt_utils.pm

```diff
@@ -306,17 +306,12 @@ sub clean_up_red_disks {
 
 sub lpar_cmd {
     my ($cmd, $args) = @_;
+    my $timeout = $args->{timeout} // 300;
```

Comment:
A bit confused about this usage. Could it be better with:

$args{timeout} //= 300 ???

Reply:
Good question and suggestion, I have the same idea on this. 
`$timeout = $args->{timeout} // 300;`
This sets $timeout to the value of **$args->{timeout}** if it is defined; otherwise, it uses 300. Importantly, it does not modify **$args->{timeout}**.

`$timeout = $args->{timeout} //= 300;`
This uses the defined-or assignment operator (//=). It first checks if `$args->{timeout} `is defined. If it isn’t, it assigns 300 to `$args->{timeout}` and then sets `$timeout `to that value. So, in addition to setting `$timeout`, it updates` $args->{timeout} `to 300 if needed.

Depend on current code logic, use with `$timeout = $args->{timeout} // 300; `better suited for automated testing of s390x kvm

Reply:
It makes no difference whether `$args` is changed here (via `//=`) because it doesn't seem to be passed to another function. So I would keep it as `my $timeout = $args->{timeout} // 300;` to avoid the unnecessary modification.